test/Test/Lorentz/Contracts/Stablecoin.hs

```diff
@@ -0,0 +1,46 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+module Test.Lorentz.Contracts.Stablecoin
+  ( test_stablecoin_contract
+  , test_setPauseAndSetTransferable
+  ) where
+
+import System.FilePath ((</>))
+import Test.Tasty (TestTree)
+import Test.Tasty.HUnit (testCase)
+
+import Lorentz.Test
+import Michelson.Test
+import qualified Michelson.Typed as T
+import Tezos.Core
+
+import Lorentz.Contracts.Stablecoin
+
+initialStorage :: Storage
+initialStorage = ()
+
+test_stablecoin_contract :: IO [TestTree]
+test_stablecoin_contract =
+  testTreesWithTypedContract "contract.tz" $
+    \(stablecoinContract :: T.FullContract (T.ToT Parameter) (T.ToT Storage)) -> pure
+  [ testCase "transfer" $ integrationalTestExpectation $ do
+      validate $ Right expectAnySuccess
+  ]
+
+test_setPauseAndSetTransferable :: IO [TestTree]
```

Comment:
It's a leftover, isn't it? I think we shouldn't test `Pause` in this issue (it's not in FA2).

Reply:
Yes. Sorry about that. Removed.